test/extended/util/db_image_helpers.go

```diff
@@ -157,13 +157,21 @@ func WaitForQueryOutput(oc *CLI, d Database, timeout time.Duration, admin bool,
 		}
 		return false, nil
 	})
+	if err == wait.ErrWaitTimeout {
+		return fmt.Errorf("Timed out waiting for query: %q", query)
```

Comment:
then add String method to that error type
On Nov 12, 2015 1:40 PM, "Martin Nagy" <notifications@github.com> wrote:

> In test/extended/util/db_image_helpers.go
> <https://github.com/openshift/origin/pull/5872#discussion_r44652105>:
>
> > @@ -157,13 +157,21 @@ func WaitForQueryOutput(oc *CLI, d Database, timeout time.Duration, admin bool,
> >  		}
> >  		return false, nil
> >  	})
> > +	if err == wait.ErrWaitTimeout {
> > +		return fmt.Errorf("Timed out waiting for query: %q", query)
>
> because its sometimes very hard to deduce what exactly happened, this
> makes things a lot easier
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5872/files#r44652105>.
>